app/models/name.rb

```diff
@@ -2219,12 +2102,64 @@ def self.new_name(params)
     result
   end
 
+  # Make a Name instance from a ParsedName
+  # Used by NameController#create_new_name
+  # Returns a Name instance, *UNSAVED*!!
+  def self.new_name_from_parsed_name(parsed_name)
+    new_name(parsed_name.params)
+  end
+
+  # Return extant Names matching a desired new Name
+  # Used by NameController#create_name
+  def self.names_matching_desired_new_name(parsed_name)
+    # unauthored ParsedName matches Names with or w/o authors
+    if parsed_name.author.empty?
```

Comment:
You are correct about being unable to create "_Blah_ group" if "_Blah_ group Author" already exists.  The question is whether that's the right behavior.

I did it that way because I thought on the create side, it should behave like non-Group Names for the same reason:  When editing a Name to remove an author we know the user has made a deliberate choice with respect to author, but when creating a Name it's more probable that the user simply overlooked the author.

I'm not sure that's the right thing to do, partly for the reason you pointed out.  What do you think? Should we ask other people to weigh in?